<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taz Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frames">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="vertical">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>8</number>
           </property>
           <property name="rightMargin">
            <number>8</number>
           </property>
           <item>
            <widget class="QSlider" name="sliderfilm">
             <property name="statusTip">
              <string/>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>6</number>
              </property>
              <property name="topMargin">
               <number>6</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>6</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="leftMargin">
                 <number>8</number>
                </property>
                <property name="rightMargin">
                 <number>8</number>
                </property>
                <item>
                 <widget class="QPushButton" name="decreaseRate">
                  <property name="toolTip">
                   <string>Decrease Play Speed</string>
                  </property>
                  <property name="statusTip">
                   <string>Decrease Play Speed</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="increaseRate">
                  <property name="toolTip">
                   <string>Increase Play Speed</string>
                  </property>
                  <property name="statusTip">
                   <string>Increase Play Speed</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="open">
                  <property name="toolTip">
                   <string>Open Video</string>
                  </property>
                  <property name="statusTip">
                   <string>Open Video</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="stop">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="mouseTracking">
                   <bool>false</bool>
                  </property>
                  <property name="toolTip">
                   <string>Stop</string>
                  </property>
                  <property name="statusTip">
                   <string>Stop</string>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="skipback">
                  <property name="toolTip">
                   <string>Skip Backward</string>
                  </property>
                  <property name="statusTip">
                   <string>Skip Backward</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="play">
                  <property name="toolTip">
                   <string>Play/Pause</string>
                  </property>
                  <property name="statusTip">
                   <string>Play/Pause</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="skipforward">
                  <property name="toolTip">
                   <string>Skip Forward</string>
                  </property>
                  <property name="statusTip">
                   <string>Skip Forward</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Expanding</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QSlider" name="volume">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="statusTip">
                   <string>Volume</string>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="listbtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>20</height>
              </size>
             </property>
             <property name="toolTip">
              <string>With this you can easily handle to jump from one part of video to another</string>
             </property>
             <property name="statusTip">
              <string>Easy Acess</string>
             </property>
             <property name="whatsThis">
              <string>With this you can handle to easy jump from one part of video to another</string>
             </property>
             <property name="text">
              <string>^</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_2" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QListWidget" name="listView">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>100</height>
                   </size>
                  </property>
                  <property name="toolTip">
                   <string>With this you can easily handle to jump from one part of video to another</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>100</width>
                    <height>100</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <layout class="QVBoxLayout" name="verticalLayout_8"/>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">blue</string>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSearch_By_Tag"/>
    <addaction name="actionFullscreen"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="theme1"/>
    <addaction name="theme2"/>
    <addaction name="theme3"/>
    <addaction name="theme4"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSearch_By_Tag">
   <property name="text">
    <string>Search By Tag</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen/normalscreen</string>
   </property>
  </action>
  <action name="theme1">
   <property name="text">
    <string>Theme 1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="theme2">
   <property name="text">
    <string>Theme 2</string>
   </property>
  </action>
  <action name="theme3">
   <property name="text">
    <string>Theme 3</string>
   </property>
  </action>
  <action name="theme4">
   <property name="text">
    <string>Theme 4</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
